src/main/integrations/electron-events.ts

```diff
@@ -75,7 +75,7 @@ export class ElectronEvents implements Integration {
         const breadcrumb = {
           category: 'electron',
           message: `${category}.${event}`,
-          timestamp: new Date().getTime() / 1000,
+          timestamp: new Date().getTime() / 1_000,
```

Comment:
Aren't numeric seperators Node 12+?

Reply:
Something like that, however we're using TypeScript targeting ES6 so these get transpiled down to ES6 compatible JavaScript!